DTDL/edge/syslog/syslog-source.json

```diff
@@ -0,0 +1,63 @@
+{
+    "@context" : "dtmi:dtdl:context;2",
+    "@id" : "dtmi:io:tributech:source:syslog;1",
+    "@type": "Interface", 
+    "extends": "dtmi:io:tributech:source:edge:base;1",
+    "displayName": "Syslog Source",
+    "contents" : [
+        {
+            "@type": "Property",
+            "name": "ListenerUseSSL",
+            "writable": true,
+            "displayName": "Listener use SSL",
+            "description": "Listener type",
+            "schema": "boolean"
+        },
+        {
+            "@type": "Property",
+            "name": "ListenerSSLCertificate",
```

Comment:
Whats the use of this property ? We will always need a certificate if we switch on the ssl right ? And the certificate must be provided on the vm / edge device and not via the twin configuration or do i miss something here ?